### PR TITLE
Update ingress.tf

### PR DESCRIPTION
**Pull request by peache41**
#11 
Update ingress.tf to use networking/v1

### CODE REVIEW

File: ingress.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resource "kubernetes_ingress" "ingress" {
```
[the remark this answers is not in the document]
Comment (patoarvizu) [reply ingress.tf:1]: Oh nice!

I see you pushed another change earlier today. Do you have any more changes coming, or is the PR ready to be reviewed?

[the remark this answers is not in the document]
Comment (peache41) [reply ingress.tf:1]: No, that would be all. Thank you

[the remark this answers is not in the document]
Comment (patoarvizu) [reply ingress.tf:1]: Sounds good! I'll approve and merge this now, and I'll do the proper release as soon as I can.

Thank you!

**Review by patoarvizu — APPROVED** [review]
👍

**Comment by patoarvizu** [comment]
Release published! https://github.com/patoarvizu/terraform-kubernetes-notary/releases/tag/v0.1.0

**Pull request merged**